website/addons/googledrive/utils.py

```diff
@@ -47,7 +47,7 @@ def log(self, action, extra=None, save=False):
         params = {
             'project': self.node.parent_id,
             'node': self.node._primary_key,
-            'folder': self.node.get_addon('googledrive', deleted=True).folder_path
+            'folder': self.node.get_addon('googledrive', deleted=True).folder_name
```

Comment:
Would it be possible to maintain consistency of the logs' display without having to change the way we store logs here?